optional_requirements.txt

```
@@ -1,3 +1,5 @@
 # SPDX-FileCopyrightText: 2022 Alec Delaney, for Adafruit Industries
 #
 # SPDX-License-Identifier: Unlicense
+
+adafruit-circuitpython-sdcard from adafruit-circuitpython-sd
```

This should just be:

```suggestion
adafruit-circuitpython-sd
```